### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by piotrtobolski** [TheTimeWalker/wallpanel-android#160] [General]
This is a great tool but I see it wasn't updated for a long time. Can it be considered dead? There is a lot of ideas and bug reports in the Pull requests tab but it doesn't seem that any of them was answered in the last year or so.

**Comment by L1nKinc** [comment]
Yeah, unfortunately I think the project is no longer maintained like it should. The app has a lot of potential I think, but the issues and pull requests are not answered in some time now. Even some promoted features do not work as intended.

Don't get me wrong, I definitely appreciate the work of @TheTimeWalker, especially as he took the project over as it was dead bevor. But maybe it is time that anyone with more time gets the maintainer of it now. 

@TheTimeWalker can you comment on this?

**Reply by sorryusernameisalreadytaken** [reply]
I think the same. This Projekt here seems to be not really maintained atm.
But this is what opensource is. There should never be a project maintained by just one person. In the reality it looks a bit different, sadly.

I have no knowledge in kotlin or any other android compatible writing language so my only chance was to use Kiosk Browser.

**Comment by e-p-s** [comment]
Is there an alternative or a fork?

(Besides FullyKioskBrowser)

**Reply by sorryusernameisalreadytaken** [reply]
Fully Kiosk-Browser cost only 10€, has a lot or let me say endless config options and it not made by a big company. So the money is in good hands.

**Reply by piotrtobolski** [reply]
Fully Kiosk is fine but in some cases this is better. It is smaller, and the costs can creep up when you have dozens of nspanels (I don't though).

It does have mqtt interface and shares changes immediately. FullyKiosk half-assed this and publishes changes to mqtt periodically which is just bad.

I want to automate displaying camera on the wall panel and I need to know exactly what is on when doorbell rings. In fully I need to poll for the current screen which makes things slow.

**Reply by sorryusernameisalreadytaken** [reply]
Oh interesting. I will check that.

I need fast mqtt reactions as well.
But did you confrontate the developer of kiosk browser with that? Just write him, he is a nice dude and this is his digital baby.

**Comment by Freddan101** [comment]
I get MQTT updates instantly with Fully Kiosk. Never had a problem.

**Reply by piotrtobolski** [reply]
![image](https://github.com/user-attachments/assets/4e88bf4f-139a-45d1-ace2-674a8598d3b2)
Only some events are push based. It explicitly says that the data is published every 60 seconds. Because of this, integration class with HA is local polling, not push/immediate.
